database/postgresSeed.js

```diff
@@ -4,32 +4,49 @@ const randomNumberUpTo = (limit) => Math.floor(Math.random() * limit);
 
 // create listings and insert in batches of 10,000
 const seedListings = async () => {
-  for (let i = 0; i < 1000; i++) {
+  let t1 = Date.now();
+  for (let i = 0; i < 20000; i++) {
     let listingsBatch = [];
-    for (let j = 0; j < 10000; j++) {
+    let bookedDatesBatch = [];
+    let batchSize = 500;
+    
+    for (let j = 0; j < batchSize; j++) {
+
+      // create listing
       let price = 50 + randomNumberUpTo(400);
       let maxguests = 1 + randomNumberUpTo(6);
       let minstay = 1 + randomNumberUpTo(2);
       let stars = (1 + (Math.random() * 4)).toFixed(2);
       let numratings = randomNumberUpTo(110);
+
+      let listing_id = (i * batchSize) + j + 1;
```

Comment:
When I used sequelize, the id was able to auto-increment but I'm not sure how it works with the modules you're using

Reply:
In my schema, id is set up to auto-increment. I could rework, but while seeding I think passing the listing id explicitly is preferable because:

- I also need the listing_id to create the listing's associated bookedDates, and it would get messier if I needed to insert the listings before getting their id's to create their bookedDates with
- We absolutely need listing_id 1 to 10,000,000